### ISSUE TITLE
Still compatible?

### ISSUE THREAD

**Issue by genofire** [mb21/iCalendar-to-Facebook-Event#4]
a few years later, after graph API?
Is it still compatible with the current facebook api?

**Comment by mb21** [comment]
Uh, don't know, haven't used it since then. But if it's not, probably wouldn't take too much work to update it :)

**Comment by jamma-schwarze** [comment]
No. I took over development from mb21 some years ago. Facebook graph API 2.0 dropped all event modification endpoints.

I just checked current Graph API (https://developers.facebook.com/docs/graph-api/reference/event/#Creating) regarding Creating, Updating or Deleting events:

> You can't perform this operation on this endpoint.

**Comment by mb21** [comment]
Thanks @jamma-schwarze 

Indeed, the people on [SO don't have a workaround either](https://stackoverflow.com/questions/23676250/facebook-graph-api-2-0-creating-events). Event creation through the API was locked down by Facebook.

**Comment by genofire** [comment]
Use the Facebook-SDK the Graph-API ?

**Comment by mb21** [comment]
@genofire yes, the SDK uses the Graph API.

**Comment by blackboxlogic** [comment]
[This ](https://developers.facebook.com/docs/pages/official-events) might help anyone who stumbled in here.
